Geometry/HGCalCommonData/plugins/dd4hep/DDHGCalTBModuleX.cc

```diff
@@ -349,8 +343,8 @@ static long algorithm(dd4hep::Detector& /* description */, cms::DDParsingContext
     zi = zo;
   }
 #ifdef EDM_ML_DEBUG
-  edm::LogVerbatim("HGCalGeom") << "DDHGCalTBModuleX: All blocks are placed in " << convertCmToMm(zMinBlock) << ":"
-                                << convertCmToMm(zi) << " with " << copies.size() << " different wafer copy numbers";
```

Comment:
In this file, your only changes are to the debug messages and `const` to `auto`. The messages will no longer convert lengths from cm to mm. Does that mean these lengths are now already in mm, or does it mean that the debug messages are reporting lengths in cm now?

Reply:
@bsunanda Could you please explain the elimination of `convertCmTMm` here? I'm not understanding the reason for the change.

Reply:
Now I put in conversion to the same unit as done in DDD. Unfortunately convertCmTMm will work for one version of dd4hep. So I use this expression